frontend/src/router/routeHandlers.ts

```diff
@@ -91,10 +91,30 @@ const beforeEnterGrantApplicationAdminRoute = async (
     return true;
 }
 
+const beforeEnterGrantDelegationAdminRoute = async (
+    to: RouteLocationNormalized
+) => {
+
+    if (!LoginUserState.hasAccess(APP_ADMIN_ROLE)) {
```

Comment:
we don't need to check the user role here, cause if we set `requiredPrivileges: [APP_ADMIN_ROLE]` in the file above, the handler `beforeEachRouteHandler` already checks for permission. 

But we do need to check if the selected application is FAM or not, same as the `beforeEnterGrantUserPermissionRoute`, when FAM is selected, the user shouldn't access to grant delegated admin form

Reply:
Thanks Catherine. I'll correct that.